src/test/java/de/id/quarkus/kafka/testing/kafkastreams/KStreamsDonatorEnrichmentTest.java

```diff
@@ -30,7 +30,7 @@
 
 @QuarkusTest
 @TestProfile(DonatorEnrichmentProfile.class)
-@QuarkusTestResource(ConfluentStack.class)
+@QuarkusTestResource(value = ConfluentStack.class, restrictToAnnotatedClass = true)
```

Comment:
we need this to avoid unpredictable tests configuration due to double starts of ConfluenceStack: https://github.com/quarkusio/quarkus/issues/22025#:~:text=apply%20this%20argument%20to%20all%20tests